Web/src/Routes.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { lazy, Suspense } from 'react';
+import { Switch, Route } from 'react-router-dom';
+
+const LandingPage = lazy(() => import('./components/landing'));
+
+const Routes: React.FC = () => (
+  <Suspense fallback={<div>Loading...</div>}>
```

Comment:
Not sure if you wanna stylize the fallback

Reply:
Haha I'm intending to to that in the next PR :)

Reply:
👍